.github/workflows/asv.yml

```diff
@@ -82,13 +82,43 @@ jobs:
 
     - name: Setup asv
       run: |
-        bash benchmarks/regression/merge_history.sh
-        cp merge_history.txt benchmarks/regression/
-        asv machine --config benchmarks/regression/asv.conf.json --yes
+        asv machine --config benchmarks/regression/asv.conf.json --machine F8s_v2 --os ubuntu-18.04 --arch x86-64 --cpu IntelXeon8168 --num_cpu 8 --ram 16GB
 
     - name: Run benchmarks
       run: |
-        asv run -v --show-stderr --config benchmarks/regression/asv.conf.json HASHFILE:merge_history.txt --cpu-affinity 0-7
+        asv run -v --show-stderr --config benchmarks/regression/asv.conf.json --cpu-affinity 0-7 --machine F8s_v2
+
+    - name: Stash benchmarks results
+      uses: actions/upload-artifact@v2
+      with:
+        name: asv-results
+        path: /benchmarks/regression/.asv/results/
+
+    - name: Checkout asv-results branch
+      uses: actions/checkout@v1
+      with:
+        ref: asv-results
```

Comment:
I mean, `clean: false` here and no need for artefacts maybe? mmhhh

Reply:
If that's what ti does even better yes